### DISCUSSION TITLE
Does it make sense to differentiate by version?

### DISCUSSION THREAD

**Discussion by balanza** [pagopa/technology-radar#19] [Ideas]
As I mentioned in [this comment](https://github.com/pagopa/technology-radar/pull/14#pullrequestreview-1759498313), does it make sense to specify which version of the tool we are referring to?

Example: fp-ts v2

**Comment by gunzip** [comment]
I think that including the specific version of a tool may be beneficial whenever the version change brings significant improvements or changes that might affect how the tool is used.
